packaging/technical-addon/internal/packaging/targz.go

```diff
@@ -0,0 +1,64 @@
+package packaging
+
+import (
+	"archive/tar"
+	"compress/gzip"
+	"io"
+	"os"
+	"path/filepath"
+)
+
+func PackageAddon(sourceDir, outputFile string) error {
```

Comment:
I think this is not being used right now... anyway, why do you think this needs to be implemented directly? Portability?